packages/contracts/deployments/README.md

```diff
@@ -8,19 +8,22 @@
 *The contracts relevant for the majority of developers are `OVM_ETH` and the cross-domain messengers. The L2 addresses don't change.*
 
 ### Predeploy contracts:
-| Contract                    | Address                                      |
-| --------------------------- | -------------------------------------------- |
-| OVM_ETH:                    | `0x4200000000000000000000000000000000000006` |
-| OVM_L2StandardBridge:       | `0x4200000000000000000000000000000000000010` |
-| OVM_L2CrossDomainMessenger: | `0x4200000000000000000000000000000000000007` |
-| OVM_L2ToL1MessagePasser:    | `0x4200000000000000000000000000000000000000` |
-| OVM_L1MessageSender:        | `0x4200000000000000000000000000000000000001` |
-| OVM_DeployerWhitelist:      | `0x4200000000000000000000000000000000000002` |
-| OVM_ECDSAContractAccount:   | `0x4200000000000000000000000000000000000003` |
-| OVM_SequencerEntrypoint:    | `0x4200000000000000000000000000000000000005` |
-| Lib_AddressManager:         | `0x4200000000000000000000000000000000000008` |
-| OVM_SequencerFeeVault:      | `0x4200000000000000000000000000000000000011` |
-| ERC1820Registry:            | `0x1820a4B7618BdE71Dce8cdc73aAB6C95905faD24` |
+|Contract|Address|
+|--|--|
+|OVM_L2ToL1MessagePasser|0x4200000000000000000000000000000000000000|
+|OVM_L1MessageSender|0x4200000000000000000000000000000000000001|
+|OVM_DeployerWhitelist|0x4200000000000000000000000000000000000002|
+|OVM_ECDSAContractAccount|0x4200000000000000000000000000000000000003|
+|OVM_SequencerEntrypoint|0x4200000000000000000000000000000000000005|
+|OVM_ETH|0x4200000000000000000000000000000000000006|
+|OVM_L2CrossDomainMessenger|0x4200000000000000000000000000000000000007|
+|Lib_AddressManager|0x4200000000000000000000000000000000000008|
+|OVM_ProxyEOA|0x4200000000000000000000000000000000000009|
+|OVM_ExecutionManagerWrapper|0x420000000000000000000000000000000000000B|
+|OVM_GasPriceOracle|0x420000000000000000000000000000000000000F|
+|OVM_SequencerFeeVault|0x4200000000000000000000000000000000000011|
+|OVM_L2StandardBridge|0x4200000000000000000000000000000000000010|
+|ERC1820Registry|0x1820a4B7618BdE71Dce8cdc73aAB6C95905faD24|
```

Comment:
It would be nice if we could maintain the Markdown prettification (i.e., indenting and fixed table width)

Reply:
Does this get prettified with `eslint`? Or is there a different JS tool that can automatically prettify it?

Reply:
FWIW, it looks like the prettification was done last time by some unknown linter

https://github.com/ethereum-optimism/optimism/pull/1119/files/61b1e5af4e261a7e58eb65131bd654886eb8283e#r657414256

Reply:
I think `prettier` is used for the linting, perhaps `yarn lint`?

Reply:
I tried `yarn lint` and it didn't prettify the markdown file. I went ahead and just ran prettier on the file directly using the CLI.